File: Server/src/GHR.Domain/Meta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GHR.Domain
{
    public class Meta
    {
        public int Id { get; set; }
        public int SupervisorId { get; set; }
        public string NomeMeta { get; set; }
        public string Descricao { get; set; }
        public Boolean MetaCumprida { get; set; }
        public Boolean MetaAprovada { get; set; }
        public DateTime? InicioPlanejado { get; set; }
        public DateTime? FimPlanejado { get; set; }
        public DateTime? InicioRealizado { get; set; }
        public DateTime? FimRealizado { get; set; }
        public IEnumerable<FuncionarioMeta> FuncionariosMetas { get; set; }
    }
}

File: SPA/GHR-App/src/app/models/Meta.ts
import { Funcionario } from "./Funcionario";

export interface Meta {
  id: number;
  supervisorId: number;
  nomeMeta: string;
  descricao: string;
  metaCumprida: boolean;
  metaAprovada: boolean;
  inicioPlanejado?: string;
  fimPlanejado?: string;
  inicioRealizado?: string;
  fimRealizado?: string;
  funcionarios: Funcionario[];
}
